lib/test.js

```diff
@@ -192,11 +203,11 @@ Test.prototype.exit = function () {
 		this.ended = true;
 
 		setImmediate(function () {
```

Comment:
Wonder if we should use https://github.com/sindresorhus/immediate-promise and have `.exit` return a promise. Related to: https://github.com/sindresorhus/ava/pull/240/files#r45325153